tests/integration/ceph_base_object_test.go

```diff
@@ -279,12 +281,26 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 			_, poErr = s3client.PutObjectInBucket(bucketname, ObjBody, ObjectKey3, contentType)
 			assert.Error(s.T(), poErr)
 		})
-
+		t.Run("test update quota on OBC bucket", func(t *testing.T) {
+			poErr := helper.BucketClient.UpdateObc(obcName, bucketStorageClassName, bucketname, newMaxObject, true)
+			assert.Nil(s.T(), poErr)
+			updated := utils.Retry(5, 2*time.Second, "OBC is created", func() bool {
```

Comment:
Should this message be changed?

```suggestion
			updated := utils.Retry(5, 2*time.Second, "OBC is updated", func() bool {
```